front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```diff
@@ -97,4 +97,17 @@
       --dsg-cell-disabled-background-color: var(--ambientB5);
     }
   }
+
+  .align-right-responsive {
+    text-align: right;
+    span {
+      width: 100%;
```

Comment:
Since the parent is `display: flex`, I think `flex-grow: 1` would be more appropriate here.

Since the `span` takes up the whole width, I don't think we need to change the class of the parent? In other words, we could set the alignment on the `span` instead?